packages/yourloops/test/integration/mock/data.api.mock.ts

```diff
@@ -81,7 +82,250 @@ const yesterdayDateAsString = YESTERDAY_DATE.format('YYYY-MM-DD')
 
 export interface Data {
   dataRange: PatientDataRange
-  data: PatientData
+  data: MedicalData
+}
+
+const addMilliseconds = (time: string, milliseconds: number) => {
+  const mTime = moment.parseZone(time)
+  return moment.utc(mTime).add(milliseconds, 'milliseconds').toISOString()
+}
+
+const getWeekDay = (date) => {
+  const days = ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', "saturday"]
+  return days[date.getDay()]
+}
+
+const convertCbgMg2Mmol = (mgValues: Cbg[]) => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** It looks like the methods `convertCbgMg2Mmol()` and `convertSmbgMg2Mmol()` are doing the same thing, could they be factorized?‏

Reply:
corrected